app/frontend/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,29 @@
+.banner {
+  height: 100vh;
+  background-image: url("../../images/photos/banner.jpg");
+  background-size: cover;
+  background-position: center -260px;
+
+  .text {
+    display: grid;
+    grid-auto-flow: row;
+    grid-row-gap: 60px;
+
+    h1 {
+      max-width: 280px;
+      font-size: 40px;
+      padding: 5px;
+      color: #fff;
+    }
+
+    p{
+      font-size: 15px;
+      font-weight: bold;
+      color: #fff;
+    }
+
+    .btn {
+      max-width: 100px;
+    }
+  }
```

Comment:
This text's position doesn't look right on my 1440p screen
Can you make it responsive to always be in the middle left please ?
![image](https://github.com/wJoenn/YogaUnalome/assets/75388869/094f30a6-c0c6-4045-829b-1416aa28701a)